README.md

```diff
@@ -1 +1,83 @@
-# todo-list
\ No newline at end of file
+# TODO LIST
+
+> The purpose of this project is to build a todo list app in order to showcase our mastery over object oriented design in javascript. 
+  
+## :hammer: Built With
+
+- HTML5
+- Javascript ES6
+- Webpack
+- Bulma
+
+## Live Demo
+
+[Todo App](https://livedemo.com)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+  
+  You need to have git installed on your local system.
+  You can clone this project with the command `$ git clone https://www.github.com/mariobarrioss/todo-list.git`.
+  For this project you also require NPM installed on your machine.
+  
+### Setup
+  
+  No Further setup is required beyond cloning the repo on your local machine.
+  
+### Install
+  
+  To install the dependencies for the application run the following commands in your terminal
+  
+  `$ npm install`
+  
+### Usage
+  
+  To build the appplication the following command needs to be run in the terminal
+  
+  `$ npm run buiid`
+  
+### Run tests
+  
+  There are no tests for this project.
+  
+### Deployment
+  
+  This section will be updated later.
+
+
+## Authors
+
+👤 **Rohan Sundar**
+
+- GitHub: [@rsundar](https://github.com/rsundar)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)

```

Comment:
## Remove/ADD actual links.
- [ ] Please avoid using placeholders for links not being used. We are welcomed to either add actual Twitter handles or rid these links of the readme.